google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -636,7 +640,10 @@ private static final class OutstandingPublish {
     OutstandingPublish(PubsubMessageWrapper messageWrapper) {
       this.publishResult = SettableApiFuture.create();
       this.messageWrapper = messageWrapper;
-      this.messageSize = messageWrapper.getPubsubMessage().getSerializedSize();
+      this.messageSize =
+          CodedOutputStream.computeMessageSize(
+              PublishRequest.MESSAGES_FIELD_NUMBER,
+              messageWrapper.getPubsubMessage().message);
```

Comment:
I accidentally caused a compilation error here when fixing a merge conflict. This should just be `messageWrapper.getPubsubMessage()`instead of `messageWrapper.getPubsubMessage().message`. My apologies for this isuse.